blockmanager.go

```diff
@@ -2620,6 +2620,13 @@ func (b *blockManager) handleHeadersMsg(hmsg *headersMsg) {
 					hmsg.peer.Disconnect()
 					return
 				}
+				// Make sure headers being processed are connecting
```

Comment:
nit: can you please add a newline before the comment and after your added block of code? Also, sentences should always be complete, including punctuation.

Reply:
Could you assist?  I do not have a working dev env.

Reply:
What do you mean by assist? Take over the PR? I can't push changes to your branch...

Reply:
Yes, that's fine!